.github/workflows/lint.yml

```
@@ -12,8 +12,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v5
         with:
```

actionlint (1.7.12)

[error] "with" section should not be empty. please remove this section if it's unnecessary

(syntax-check)